samples/AddNamespaceResourceQuota.md

```diff
@@ -8,28 +8,28 @@ To limit the number of resources like CPU and memory, as well as objects that ma
 
 ## Policy YAML 
 
-[require_namespace_quota.yaml](best_practices/require_namespace_quota.yaml) 
+[add_ns_quota.yaml](best_practices/add_ns_quota.yaml) 
 
 ````yaml
 apiVersion: kyverno.io/v1alpha1
 kind: ClusterPolicy
 metadata:
-  name: generate-namespace-quota
+  name: add-ns-quota
 spec:
   rules:
-  - name: generate-namespace-quota
+  - name: generate-resourcequota
     match:
       resources:
         kinds:
         - Namespace
     generate:
       kind: ResourceQuota
-      name: "defaultresourcequota"
+      name: "default-resourcequota"
       data:
         spec:
           hard:
             requests.cpu: '4'
             requests.memory: '16Gi'
-            limits.cpu: '4'
-            limits.memory: '16Gi'
+            limits.cpu: $(../../requests/cpu)
```

Comment:
Is this valid syntax in Kubernetes? Tested and got error in policyviolations:
```
message: 'unable to create resource ResourceQuota/quota/default-resourcequota:
      ResourceQuota in version "v1" cannot be handled as a ResourceQuota: v1.ResourceQuota.Spec:
      v1.ResourceQuotaSpec.Hard: unmarshalerDecoder: quantities must match the regular
      expression ''^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'', error found in
      #10 byte of ...|ests/cpu)","limits.m|..., bigger context ...|pec":{"hard":{"limits.cpu":"$(../../requests/cpu)","limits.memory":"$(../../requests/memory)","reque|...'
```

Reply:
We may have to add quotes?

Reply:
looks like we do not allow variables in the "generate" rule...

will update.